README.md

```diff
@@ -1 +1,34 @@
-# employees-management-ui
\ No newline at end of file
+# Kainos Recruitment UI node.js repo
+This is the repository containing the node.js project that runs the Kainos Recruitment application's UI.
+
+---
+## Requirements
+For development and testing, you will need Node.js installed.
+
+### Installing Node
+You can find more information about the installation for specific operating systems on the [official Node.js website](https://nodejs.org/) and the [official NPM website](https://npmjs.org/).
+
+If the installation was successful, you should be able to run the following command.
+
+    $ node --version
+
+    $ npm --version
+
+### Building and running the project
+You can build this node.js application simply by running the following command.
+
+    $ npm install
+
+Once all of the dependencies have finished installing, you can start the application by running the following command.
+
+    $ npm start
+
+### Testing
+You can run the tests contained in the project's test folder simply by running the following command.
+
+    $ npm test
```

Comment:
npm test will only run the UI tests. The command I gave you will run all the tests including the automation ones

Reply:
Ah no worries, will edit that now